README.md

```diff
@@ -18,12 +18,22 @@ Prior to starting the service locally, make sure the instance running in service
 ### From source code on your local machine
 1. Clone this repository into the development environment on your machine.
 2. Open a terminal and navigate to the folder you cloned the service into.
-3. Run either `./run.sh` or `sbt "run 9590 -Dapplication.router=testOnlyDoNotUseInAppConf.Routes` to start the service locally.
+3. Run either `./run.sh` or `sbt "run 9590" -Dapplication.router=testOnlyDoNotUseInAppConf.Routes` to start the service locally.
 4. Ensure all dependent services are started, using the `SOFTWARE_CHOICES_ALL` service manager profile.
 5. In a browser, navigate to `http://localhost:9590/making-tax-digital-software`.
 
 This service can be used by both JavaScript users and non-Javascript users. If you want to test the service with no JavaScript enabled, disable it in your browser site settings and reload the page.
 
+### Feature flags and their UI labels
+Each feature flag in the configuration controls a specific section or functionality on the UI, as shown below:
+1. `business`: Businesses
+2. `agent`: Agents
+3. `accounting`: VAT record keeping
+4. `spreadsheets`: Bridging
```

Comment:
good job on adding labels - If possible add other labels as well like free/ welsh etc..

![image](https://github.com/user-attachments/assets/9cb349fe-d03b-414a-b020-428adbf74160)

Reply:
they're quite obvious, that's why i didn't add them. like free = free, blindness = visual, etc. do you still want me to add them?